Python/Distance-Calculator-App/main.py

```diff
@@ -0,0 +1,22 @@
+from geopy.geocoders import Nominatim
+from geopy import distance
+import argparse
+def location():
+    geolocator = Nominatim(user_agent="geoapiExercises")
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--firstlocation', type=str, required=True)
+    parser.add_argument('--secondlocation', type=str, required=True)
+    args = parser.parse_args()
+    x = geolocator.geocode(args.firstlocation)
```

Comment:
Instead of naming `x` and `y` for two distinct locations, name meaningful variables as `first_location` and `second_location`.